### ISSUE TITLE
Installed but ... nothing. Just me?

### ISSUE THREAD

**Issue by dajare** [vjt/afloat#7]
I picked up this fork from [AskDifferent](http://apple.stackexchange.com/a/192003/136150) and was happy to find it. I'm an Ubuntu/Mint user for whom "pinning" a window to stay on top is done natively and simply.

However, having followed the install instructions, I don't seem to have any additonal "window" options. It does look as if it is installed:

![simba window](http://i.imgur.com/7ZbeqV3.png)

and (after a re-start, just in case) I have Simba "running" in the dock and minimized. However, no application I've tried (FF, Pages, Notes, whatever) have any additonal options under the "Window" menu, nor anywhere else I can find.

Am I missing a crucial step? I'm on a MacBook (Early 2015) running Yosemite 10.10.5.

**Comment by tyilo** [comment]
See https://github.com/norio-nomura/EasySIMBL/issues/26

tl;dr: EasySIMBL is broken, use SIMBL instead.

**Comment by sebastienkb** [comment]
Hi guys, just a quick note to say that Afloat didn't work for me on El Capitan despite the many instructions available on Afloat/EasySIMBL/SIMBL. The only thing that worked for me is disabling El Capitan's System Integrity Protection (SIP) (read from http://www.chrisvanpatten.com/master-windows-afloat-mac#comment-2299581913).
Instructions that worked for me:
1. Reboot
2. Hold Cmd+R while booting (Recovery mode)
3. Go to Utilities menu and select Terminal
4. Type `csrutil disable; reboot`, your Mac will reboot normally
5. Open EasySIMBL and uncheck/recheck "Use SIMBL" and reboot again

Unfortunately I tried re-enabling SIP with `csrutil enable; reboot` after but that disabled Afloat again. So it's up to you if you wish to keep SIP or not.
